packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -456,6 +456,7 @@ <h2 class="demo-title">Swap textures - Normals/Occlusion/Emission</h2>
         <option value="../../shared-assets/models/glTF-Sample-Models/2.0/WaterBottle/glTF/WaterBottle_normal.png">Water Bottle</option>
       </select>
     </div>
+  </div>
```

Comment:
Thanks! Technically we could probably just remove the useless `<div>` a few lines up instead of adding this...

Reply:
Done.

Reply:
the full test run takes a long time it's normal ?

Reply:
No, that one doesn't run for external contributors. I should probably try to update github actions. Thanks for the ping.